addons/payment_stripe/models/payment_transaction.py

```diff
@@ -54,15 +54,24 @@ def _stripe_create_checkout_session(self):
         :return: The Checkout Session
         :rtype: dict
         """
+        def get_linked_pmts():
+            linked_pmts_ = linked_pms
+            card_pms = [
+                self.env.ref(f'payment.payment_icon_cc_{pm_code_}')
```

Comment:
Could be better to avoid strict refs and use the names (or consider using raise_if_not_found)

Reply:
I originally intended to force the user to fix their broken XMLIDs, but after reconsideration, it should still work if only one out of the four card icons still exists.